frontend/src/lib/components/PropertyFilters/PropertyFilters.stories.tsx

```diff
@@ -36,6 +36,37 @@ export function ComparingPropertyFilters(): JSX.Element {
     useStorybookMocks({
         get: {
             '/api/event/values/': [],
+            '/api/projects/:team_id/property_definitions*': {
```

Comment:
this is a bit of a doozy, because without knowing the property type, our operator borks and goes to the default.

Reply:
I'm not sure how big of a problem this is in production? Do property_definitions requests fail often? If yes, this 'improvement' makes things worse for the default case 😬 .

Under the assumption this request succeeds, things are smooth

Reply:
They don't fail (and if they did, we should just fix that ASAP), but they can take a while. For us, with lots of properties, that's about TWELVE seconds, having checked just now. So we should also have sane default behavior in case the property definitions haven't been loaded yet
(And of course this endpoint _should_ always be fast, but slowness is not urgent in this case)

Reply:
This flow should be fine I think: Before loading succeeds, we show the default list (no custom date / numeric operators). After loading, we can show the 'right' list.

For trying to edit existing properties with a custom date operator, this indeed borks and shows 'equals' instead of the raw unknown operator if property definitions haven't loaded.

12 seconds is long enough, so for the latter case I'll check if the definitions have loaded, and only switch operator if they have.